### PR TITLE
Add package_readline_6_3

### PR DESCRIPTION
**Pull request by blankenberg**

### CODE REVIEW

File: packages/package_readline_6_3/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<tool_dependency>
+     <package name="ncurses" version="6.0">
+        <repository name="package_ncurses_6_0" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="readline" version="6.3">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">ftp://ftp.gnu.org/gnu/readline/readline-6.3.tar.gz</action>
```
Comment (erasche) [inline packages/package_readline_6_3/tool_dependencies.xml]: would you mind adding an sha256sum="..." for this?

**Comment by erasche** [comment]
LGTM. 

CC @malex didn't you have trouble with the readline build process? If so, I don't suppose I could get you to test this whenever we merge it?

**Comment by davebx** [comment]
Looks solid :+1:

**Comment by bgruening** [comment]
Looks, good to me. But would be nice to add the checksum.

**Comment by Malex** [comment]
@erasche ehm, I dont know hot say that but.. do we know each other? I have no idea what this repository is o.O

**Comment by erasche** [comment]
@Malex sorry, wrong user! My apologies!

**Comment by moskalenko** [comment]
Absolutely!

**Comment by bgruening** [comment]
Thanks @blankenberg!

**Pull request merged**